src/app/components/LinkedData/index.tsx

```diff
@@ -111,12 +111,37 @@ const LinkedData = ({
 
   const hasByline = !!bylineLinkedData;
 
-  const { authorName, authorTopicUrl, twitterLink, authorImage, location } =
-    bylineLinkedData || {};
+  const places: string[] = [];
 
-  const sameAs = [authorTopicUrl, twitterLink].filter(Boolean);
+  const getAuthorTagging = (data: BylineLinkedData | null) => {
+    const { authorName, authorTopicUrl, twitterLink, authorImage, location } =
+      data || {};
 
-  const locationCreated = { '@place': location };
+    const sameAs = [authorTopicUrl, twitterLink].filter(Boolean);
+    if (location) {
+      if (!places.includes(location)) {
+        places.push(location);
+      }
+    }
+
+    return {
+      '@type': 'Person',
+      name: authorName,
+      ...(sameAs.length && { sameAs }),
+      ...(authorImage && { image: authorImage }),
+    };
+  };
+  let bylineAuthors;
+  if (bylineLinkedData) {
+    bylineAuthors =
+      bylineLinkedData?.length > 1
+        ? bylineLinkedData?.map(data => getAuthorTagging(data))
+        : getAuthorTagging(bylineLinkedData?.[0]);
+  }
+
+  const locationCreated = {
+    '@place': places.length === 1 ? places[0] : places,
```

Comment:
What are we expecting the result to be here? If there's only 1 place - string, but if there's multiple, array of strings? Or is it a comma-separated list? 

(Aside, I am not convinced that this is actually the correct linked data structure - https://schema.org/locationCreated - so I am wondering if we perhaps only list 1 location in SEO for now, and perhaps revisit what it should be if there are muiltiple locations?)

Reply:
Going off this [older ticket](https://jira.dev.bbc.co.uk/browse/WSTEAM1-153) , 

> If there are multiple bylines, then we should take all the unique location values, and if there are more than one, render them as an array

Running the schema thats generated through a [markup validator](https://validator.schema.org/), or this [validator](https://search.google.com/test/rich-results) it doesnt get flagged as invalid

Reply:
OK - if it works, then let's leave as is!